src/Riok.Mapperly/Descriptors/MappingBuilders/EnumToStringMappingBuilder.cs

```diff
@@ -18,6 +20,47 @@ public static class EnumToStringMappingBuilder
 
         // to string => use an optimized method of Enum.ToString which would use slow reflection
         // use Enum.ToString as fallback (for ex. for flags)
-        return new EnumToStringMapping(ctx.Source, ctx.Target, ctx.SymbolAccessor.GetAllFields(ctx.Source));
+        return BuildEnumToStringMapping(ctx);
+    }
+
+    private static EnumToStringMapping BuildEnumToStringMapping(MappingBuilderContext ctx)
+    {
+        var ignoredSourceMembers = ctx.Configuration.Enum.IgnoredSourceMembers.ToHashSet(SymbolTypeEqualityComparer.FieldDefault);
+        var explicitMappings = BuildExplicitValueMappings(ctx);
+
+        ctx.AddUnmatchedSourceIgnoredMembers(ignoredSourceMembers);
```

Comment:
Doesn't this just always report all ignored members?

Reply:
Yes, exactly. I have corrected it to ensure that it doesn't always report all ignored members.